android/src/main/java/org/reactnative/maskedview/RNCMaskedViewPackage.java

```diff
@@ -22,4 +22,9 @@
             new RNCMaskedViewManager()
       );
   }
+  
+  @Override
```

Comment:
Are you try to run these changes on your side? `@Override` annotation should be removed.